docs/2.build/1.chain-abstraction/multichain-gas-relayer/relayer-gas-example.md

```diff
@@ -11,24 +11,34 @@ In this article you'll learn how to run end-to-end tests on the entire Multichai
 
 :::info Required tools
 
-For this tutorial, you'll need to have installed:
+For this tutorial, you need to have installed:
 
+ - [Python](https://python.org) >=3.10
+ - [Rust](https://rust-lang.org)
  - [Multichain Relayer Server](https://github.com/near/multichain-relayer-server)
  - [Gas Station Event indexer](https://github.com/near/gas-station-event-indexer)
- - [NEAR CLI RS](https://github.com/near/near-cli-rs)
+ - [NEAR CLI RS](https://github.com/near/near-cli-rs): Make sure to configure it with the correct network and account.
 
 :::
 
 ## Setup
 
-Before you start testing, set up your local environment and install the Relayer server, the Event indexer and NEAR CLI.
+Before you start testing, set up your local environment and install the [Relayer server](multichain-server.md), the [Event indexer](https://github.com/near/gas-station-event-indexer) and [NEAR CLI](https://github.com/near/near-cli-rs).
 
 :::info Alpha stage
 
 The Multichain Relayer solution is currently under development. Users who want to test-drive this solution should keep in mind that the product is in alpha stage, and a code audit is pending.
 
 :::
 
+## Integration test
+
+In separate terminals, you need to run the following tools:
+
+1. [Multichain Relayer server](#multichain-relayer-server), with a valid `config.toml` configuration file
+2. [Gas Station indexer](#gas-station-event-indexer), with correct values in `config.toml`
+3. [Python integration test script](#run-integration-test)
+
```

Comment:
I think we should call out that this is the best way to test that all multichain gas relayer systems are working well together and the Manual Testing section below is a good way to debug if any individual part of the system isn't working

Reply:
Fixed